.travis.yml

```diff
@@ -23,4 +23,4 @@ install:
   - composer self-update
   - composer update --prefer-dist --no-progress --no-suggest --optimize-autoloader $COMPOSER_FLAGS
 
-script: ./vendor/bin/atoum -d src/Smalot/PdfParser/Tests/ -ncc
+script: vendor/bin/phpunit --coverage-clover coverage/clover.xml
```

Comment:
I think we shouldn't generate coverage for each build.
Only one is enough and make the whole test suite faster.

You can re-use the same solution I used to define `$COMPOSER_FLAGS`

Reply:
You mean a separate build for coverage, like yours with `$COMPOSER_FLAGS`?

Reply:
Yep

Reply:
Done.